backend/main.py

```diff
@@ -1,2 +1,49 @@
-print("Hello world!")
-print("Hello world!")
\ No newline at end of file
+from fastapi import FastAPI
+from typing import List
+from pydantic import BaseModel
+from datetime import datetime
+
+
+class Task(BaseModel):
+    id: int
+    title : str
+    completed : bool
+
+class Item_todo(BaseModel):
+    id : int
+    created_at : datetime
+    title : str
+    color : str
+    task : List[Task]
+
+
+app = FastAPI()
+
+#create new todo
+@app.post("/todo", response_model=Item_todo)
+def create_todo(item:  Item_todo):
+    return item
+
+
+#Get list of todo
```

Comment:
Comments are not documentation. Use doc strings 
https://peps.python.org/pep-0257/